Shared/Samples/Generate offline map with custom parameters/GenerateOfflineMapWithCustomParametersView.CustomParameters.swift

```diff
@@ -0,0 +1,156 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension GenerateOfflineMapWithCustomParametersView {
+    struct CustomParameters: View {
+        /// The action to dismiss the sheet.
+        @Environment(\.dismiss) private var dismiss
+        
+        /// The view model for the download offline map area view.
+        @ObservedObject var model: Model
+        
+        /// The area of interest.
+        var extent: Envelope
+        
+        /// A Boolean value indicating whether the job is generating an offline map.
+        @Binding var isGeneratingOfflineMap: Bool
+        
+        /// The error shown in the error alert.
+        @State private var error: Error?
+        
+        /// The min scale level for the output. Note that lower values are zoomed further out,
+        /// i.e. 0 has the least detail, but one tile covers the entire Earth.
+        @State private var minScaleLevel = 0.0
+        
+        /// The max scale level for the output. Note that higher values are zoomed further in,
+        /// i.e. 23 has the most detail, but each tile covers a tiny area.
+        @State private var maxScaleLevel = 23.0
+        
+        /// The range for scale level values.
+        private let scaleLevelRange = 0.0...23.0
+        
+        /// The extra padding added to the extent envelope to fetch a larger area, in meters.
+        @State private var basemapExtentBufferDistance = 0.0
+        
+        /// The range for buffering the basemap extent.
+        private let basemapExtentBufferRange = 0.0...100.0
+        
+        /// A Boolean value indicating if the system valves layer should be included in
+        /// the download.
+        @State private var includeSystemValves = true
+        
+        /// A Boolean value indicating if the service connections layer should be included
+        /// in the download.
+        @State private var includeServiceConnections = true
+        
+        /// The minimum flow rate by which to filter features in the Hydrants layer,
+        /// in gallons per minute.
+        @State private var minHydrantFlowRate = 0.0
+        
+        /// The hydrant flow rate range.
+        private let hydrantFlowRateRange = 0.0...1500.0
+        
+        /// A Boolean value indicating if the pipe layers should be restricted to
+        /// the extent frame.
+        @State private var shouldCropWaterPipesToExtent = true
+        
+        var body: some View {
+            NavigationStack {
+                Form {
+                    Section(header: Text("Adjust Basemap")) {
+                        HStack {
+                            Text("Min Scale Level")
+                            Spacer()
+                            Text(minScaleLevel, format: .number.precision(.fractionLength(0)))
+                        }
+                        Slider(value: $minScaleLevel, in: scaleLevelRange, step: 1.0)
```

Comment:
```suggestion
                        VStack {
                            Text("Min Scale Level")
                                .badge(
                                    Text(minScaleLevel, format: .number.precision(.fractionLength(0)))
                                )
                            Slider(value: $minScaleLevel, in: scaleLevelRange, step: 1)
                        }
```

Consider using this instead for the slider rows. In my opinion, it is a little cleaner. Up to you, though.